##[error]Task Failed. Error: "Invalid Dsn"

On using the task with an agent installed in same machine, its showing "##[error]Task Failed. Error: "Invalid Dsn" ". The details in task were given as follows:

Zap Api URL: localhost:8080
Api key: **************

--- a/BuildTask/OwaspZapScan/classes/TaskInput.ts
+++ b/BuildTask/OwaspZapScan/classes/TaskInput.ts
@@ -50,7 +50,7 @@ export class TaskInput {
         if (this._clearSession) {
             return this._clearSession;
         }
-        return true;
+        return false;
     }
 
     /* Spider Scan */

